doc/stdlib/parallel.rst

```diff
@@ -703,6 +703,10 @@ Shared Arrays (Experimental, UNIX-only feature)
    Returns the index of the current worker into the ``pids`` vector, i.e., the list of workers mapping
    the SharedArray
 
+.. function:: localindexes(S::SharedArray)
+
+   Blah blah
```

Comment:
This was me testing the new documentation system---see the history in #12423. This will not be part of the final commit.